docs/FAQ.md

```diff
@@ -56,17 +56,15 @@ only applies for non required attributes with default values.
 
 # How are views for a result type computed?
 
-Views can be defined on a result type. If a method returns a result type with
-multiple views then
-* the service method returns an extra view along with the result and error
+Views can be defined on a result type. If a method returns a result type
+* the service method returns an extra view along with the result and error if
+  the result type has more than one view. The generated endpoint function uses
+  this view to create a viewed result type.
 * a views package is generated at the service level which defines a viewed
-  result type for each result with multiple views. This viewed result type
-  has identical field names and types but uses pointers for all fields so that
-  view specific validation logic may be generated. Constructors are generated
-  in the service package to convert a result type to a viewed result type and
-  vice versa.
-* the generated endpoint function uses the view returned by the service method
-  to create a viewed result type.
+  result type for each method result. This viewed result type has identical
```

Comment:
Do users have to deal with viewed result types directly? If not maybe I would simplify this entire section with something like:
> The view is used to validate the content of the result type client side.